core/src/main/java/jenkins/security/SecurityListener.java

```diff
@@ -69,13 +69,27 @@ protected void failedToAuthenticate(@Nonnull String username){}
      */
     protected void loggedIn(@Nonnull String username){}
 
+    /**
+     * Fired when a new user account has been created.
+     *
+     * @param username the user
```

Comment:
Might want to add a `@since TODO` javadoc line. A maintainer can go and replace those with the release version.